src/inputs/TextFieldBase.tsx

```diff
@@ -259,6 +274,8 @@ export function TextFieldBase<X extends Only<TextFieldXss, X>>(props: TextFieldB
                 ...(errorMsg && !inputProps.disabled ? fieldStyles.error : {}),
                 ...Css.if(multiline).aifs.oh.mhPx(textAreaMinHeight).$,
               }}
+              // Class name used for the grid table on row hover for highlighting
+              className="textFieldBaseWrapper"
```

Comment:
I could envision eventually needing other non-`TextFieldBase` components controlling the same "I'm hovered, so don't hover the other editable components in the row anymore", so I think it'd potentially be neat to:

* Rename `ROW_CSS_SELECTOR` to `BorderHoverParent`
* Extract `textFieldBaseWrapper` and name it `BorderHoverChild`

To have the names align to the general capability we're adding, which is that should a component have `borderOnHover=true` set, it will hook up its styles to "whatever BorderHoverParent", as long as said BHP doesn't already have a hovered BHC.